test/integration/src/test/java/io/pravega/test/integration/WatermarkingTest.java

```diff
@@ -514,6 +482,190 @@ public void progressingWatermarkWithWriterTimeouts() throws Exception {
         assertNull(nullMark);
     }
 
+    @Test
+    public void test() {
```

Comment:
(this was probably some debugging thing)

Reply:
Yes. :) I've removed it.